cf_release.py

```diff
@@ -247,5 +242,11 @@ def main():
 if __name__ == "__main__":
     try:
         main()
+    except RuntimeError as e:
+        print(f"Runtime Error: {str(e)}")
```

Comment:
Remove Runtime Error here... Python will handle that. In fact no print is needed here but feed the informational comment directly to the exception instance